Elevel/Application/Features/AudioFeatures/CreateAuditionCommand.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Threading.Tasks;
+using MediatR;
+using Elevel.Application.Interfaces;
+using AutoMapper;
+using System.Threading;
+using Elevel.Domain.Models;
+
+namespace Elevel.Application.Features.AudioFeatures
+{
+    public class CreateAuditionCommand
+    {
+        public class Request : IRequest<Response>
+        {
+            public string AudioFilePath { get; set; }
+            public DateTimeOffset CreationDate { get; set; }
+        }
+        public class Handler : IRequestHandler<Request, Response>
+        {
+            private readonly IApplicationDbContext _context;
+            private readonly IMapper _mapper;
+            public Handler(IApplicationDbContext context, IMapper mapper)
+            {
+                _context = context;
+                _mapper = mapper;
+            }
+            public async Task<Response> Handle(Request request, CancellationToken cancelationtoken)
+            {
+                request.CreationDate = DateTime.Now;
```

Comment:
please use DateTimeOffset.NowUtc
and if you set this property here why we have it in request?